aws/resource_aws_vpc_peering_connection.go

```diff
@@ -24,6 +23,12 @@ func resourceAwsVpcPeeringConnection() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 
+		Timeouts: &schema.ResourceTimeout{
```

Comment:
Added timeout configuration options and set defaults to the values in the existing code - 1 minute in all cases.